scripts/normalize-by-median.py

```diff
@@ -94,12 +94,12 @@ def normalize_by_median(input_filename, outfp, htable, paired, cutoff,
             med, _, _ = htable.get_median_count(seq)
 
             if med < desired_coverage:
-                htable.consume(seq)
                 passed_filter = True
 
         # Emit records if any passed
         if passed_length and passed_filter:
             for record in batch:
+                htable.consume(record.sequence)
```

Comment:
OK, sorry for the delay, took some time to wrap my head around this.  This is effectively changing the behavior of diginorm to no longer convert Ns to As but instead ignore k-mers in reads with Ns in them.  Do we want to do that? I am skeptical.

What is wrong with simply replacing Ns with As as was done previously?